tests/test_auth_storage.py

```diff
@@ -6,12 +6,15 @@
 import pytest
 
 from tensorbay import GAS
-from tensorbay.dataset import Dataset, Frame, FusionDataset, FusionSegment, Segment
+from tensorbay.dataset import Data, Dataset, Frame, FusionDataset, FusionSegment, Segment
 from tensorbay.exception import ResourceNotExistError
-from tensorbay.label import Classification
+from tensorbay.label import Catalog, Classification, Label
 from tensorbay.sensor import Lidar
+from tests.test_upload import CATALOG, LABEL
 from tests.utility import get_dataset_name
 
+_LOCAL_CONFIG_NAME = "HDFS_本地1"
```

Comment:
why we have Chinese character in the code?

Reply:
The name of the config is unified as this, which is proposed by the PM and set by the backend.